### ISSUE TITLE
Lua erroring when sync.lua file not found in directory.

### ISSUE THREAD

**Issue by bradenhelmer** [bradenhelmer/nvim-syncer#1]
Handling logic needs to be implemented for when a ```sync.lua``` file is not found in the current directory. Erroring in```config.lua```.

```
E5113: Error while calling lua chunk: ...l/share/nvim/lazy/nvim-syncer/lua/nvim-syncer/config.lua:8: module 'sync' not found:
        no field package.preload['sync']
cache_loader: module sync not found
stack traceback:
        [C]: in function 'require'
        ...l/share/nvim/lazy/nvim-syncer/lua/nvim-syncer/config.lua:8: in function 'get_config_file'
        ...l/share/nvim/lazy/nvim-syncer/lua/nvim-syncer/config.lua:15: in main chunk
```

**Comment by bradenhelmer** [comment]
Error occurring when sync.lua is present in parent directory and not the current.

**Comment by bradenhelmer** [comment]
Closed with d1cd1c1f8816ed45f1bfb18d52c1b040a3f406de
